config/environment.rb

```diff
@@ -1,5 +1,5 @@
-# Load the Rails @application_letter.
-require File.expand_path('../application', __FILE__)
+# Load the Rails application.
+require_relative 'application'
 
-# Initialize the Rails @application_letter.
```

Comment:
Yeah. Good catch 👍

Reply:
A special thanks to the Rails update script ;)